### DISCUSSION TITLE
is there a way to disable explore page?

### DISCUSSION THREAD

**Discussion by doomster** [usememos/memos#3448] [Q&A]
I just started hosting memos in a private server, amazing job people, its what i needed to ditch Keep. 

I want to expose it to the internet, and i am using Cloudflare zerotrust tunnels as i do for other services. I cannot see of a way to disable the explore page, and since i dont want to use public  notes at all, its kind of annoying for it to be accessible. it also shows a list of users, which for me is a nono when thinking of putting that out on the internet. 

Is there a way to completely disable the /expose page and/or hide it from plain site?

EDIT: i can actually block it from Cloudflare, as a URI, but i was wondering if there is an app way to do it.

**Comment by eMUQI** [comment]
Are you referring to the `/explore` path? My explore page doesn't show a user list when not logged in (maybe because I only have one user?). I'm curious about what your page looks like. I am just as concerned about privacy and security issues as you are.
![image](https://github.com/usememos/memos/assets/32332228/543f1301-ad44-4118-a78d-5c090ba9633b)

**Comment by doomster** [comment]
hey, sorry for the late reply. 
if you currently visit my page, you will see this:
![image](https://github.com/usememos/memos/assets/8209904/e7d29745-bc87-4a8d-8dff-7be6589bc22e)

these two demo users have been setup to showcase what i am saying. 
thats why i want the explore tab completely disabled.  cause it actually shows users.

**Comment by weado** [comment]
Hello, 
However, I recently started using Memos via Docker on a server with public access, but I would like the 'public' page not to appear, even if no notes are available. I don't want to tempt people who pass by. How can I disable access to the 'explore' and 'about' pages and only allow 'sign-in'?
Thanks a lot for the app !!!

**Comment by LaurentAtJoes** [comment]
Well, there should definitely be a way of disallowing _ANY_ page but log-in ...

I just installed memos because I'm looking for something just like this, and exposed it through a reverse proxy, but then I saw that on first connection it goes to "explore", (and that you can also access "about", even if it's less of an issue, you shouldn't)

This is kind of a show-stopper for me... Only log-in page should be accessible here (public notes are disabled, of course) 

-- EDIT -- although with multiple users set, I don't see their list on the "explore" page.
Actually, I don't see it even when logged in.
And TBH, I don't get what this page is for at all. Search doesn't work there, tags do not show

**Comment by neuro18** [comment]
Add custom CSS in Settings -> System -> Additional styles (under admin user):
```JavaScript
/* Remove Inbox folder */
#header-inbox {
  display: none;
}

/* Remove Profile folder */
#header-profile {
  display: none;
}

/* Remove Explore folder */
#header-explore {
  display: none;
}
```

**Reply by semiligneous** [reply]
What about the fact that anyone can Sign Up and create a new user/account? They can then see the posts/notes that are shared with the 'Workspace'... or is there a way to handle that problem that I am missing?

**Reply by harrischrisa** [reply]
> What about the fact that anyone can Sign Up and create a new user/account? They can then see the posts/notes that are shared with the 'Workspace'... or is there a way to handle that problem that I am missing?

Is there a way to stop a person being able to sign up?

Thanks

**Reply by MichaKersloot** [reply]
It's in the Settings -> Admin -> System: Disallow user registration.

**Comment by MichaKersloot** [comment]
Hiding the Explore doesn't work. The base url will still redirect to /explore
